### PR TITLE
Shawburn: Add Full Site Editing Support

### PR DESCRIPTION
**Pull request by Copons**
This PR basically copies the Full Site Editing overrides created for Maywood into Shawburn, and externalize the main navigation styles so that they can be easily imported into the editor as well..
It reuses the Maywood template parts, so they don't look exactly the same as the [Shawburn demo site](https://shawburndemo.wordpress.com/).

Part of https://github.com/Automattic/wp-calypso/issues/37429

### Screenshots

[Shawburn demo site](https://shawburndemo.wordpress.com/) header for reference:
Screenshot 2019-11-08 at 18 43 59

FSE header in the template part editor, right after applying this PR, without making any changes to the blocks:
Screenshot 2019-11-08 at 18 44 35

FSE header in the editor and in the front end after changing the blocks attributes to make it look like the demo:
Screenshot 2019-11-08 at 18 45 20Screenshot 2019-11-08 at 18 45 28

### Instructions

Pre-requisites:
- Build the FSE plugin with https://github.com/Automattic/wp-calypso/pull/37447
- Checkout D35209-code and sandbox the API

Then:
- Apply this PR and load the Shawburn theme into an FSE test site.
- Activate the Shawburn theme.
- Edit a page and make sure the template parts show up and look as expected.
- Preview the page and make sure the template parts show up and look as expected.

### CODE REVIEW

**Comment by apeatling** [comment]
This is awesome!

**Comment by noahtallen** [comment]
In D35247-code, we can easily add a demo site for this and get template content perfect :)

**Comment by Copons** [comment]
This is ready for review!

**Comment by Copons** [comment]
Note: rebase after https://github.com/Automattic/themes/pull/1654 merges.

File: shawburn/functions.php [hunk 1]
```diff
@@ -99,6 +99,9 @@ function shawburn_setup() {
 				),
 			)
 		);
+
+		// Enable Full Site Editing
+		add_theme_support( 'full-site-editing');
```
Comment (noahtallen) [inline shawburn/functions.php]: nit: missing space

```suggestion
		add_theme_support( 'full-site-editing' );
```

**Pull request merged**